contracts/cw-named-groups/src/msg.rs

```diff
@@ -0,0 +1,58 @@
+use std::collections::HashSet;
+
+use cosmwasm_std::Addr;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct Group {
+    pub name: String,
+    pub addresses: HashSet<String>,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct InstantiateMsg {
+    pub groups: Option<Vec<Group>>,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum ExecuteMsg {
+    Add {
+        group: String,
+        addresses: Option<Vec<String>>,
+    },
+    Remove {
+        group: String,
+        addresses: Option<Vec<String>>,
+    },
```

Comment:
What do you think about adding an execute message to change the owner of this contract? This might be useful if a DAO needs to migrate its router contract.